From 4a19f82da41e07562e719bd831a38887dfa44281 Mon Sep 17 00:00:00 2001
From: methanal <baclofen@tuta.io>
Date: Fri, 8 Sep 2023 11:26:21 +0000
Subject: [PATCH 008/142] arm64: dts: qcom: sdm845-oneplus-common: add PDT
 override values

Provide the values of the function number and interrupt status
registers for partially implemented PDT entries of some aftermarket
touch ICs.

Signed-off-by: methanal <baclofen@tuta.io>
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index 03943b4385..64983a3660 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -424,6 +424,8 @@ synaptics-rmi4-i2c@20 {
 		syna,reset-delay-ms = <200>;
 		syna,startup-delay-ms = <200>;
 
+		syna,pdt-fallback-desc = [34 41 01 01 12 01];
+
 		rmi4-f01@1 {
 			reg = <0x01>;
 			syna,nosleep-mode = <1>;
-- 
2.49.0

